content/02.author-sections.md

```diff
@@ -318,7 +318,11 @@ text = parsed["content"]
 
 
 <!-- Qiwen Hu -->
+### Qiwen Hu
 
+I'm a postdoc from [Greene Lab](http://www.greenelab.com/) at the University of Pennsylvania. My research focus on integrating different types of high-throughput sequencing data to find meaningful biological signals behind it. I developed machine learning and statistical approaches to identify regulatory elements that affect transcription and translation. I also developed machine learning-based methods to extract regulatory signals from addicted brain [@doi:10.1371/journal.pcbi.1005602], developmental tissues [@doi:10.1101/361816] and cell-type signals from single-cell datasets.
```

Comment:
Can we change this paragraph to have a newline after each sentence? In other words, replace the space after periods with a newline